templates/repo/commit_load_branches_and_tags.tmpl

```diff
@@ -0,0 +1,13 @@
+<div class="branch-and-tag-area">
+	<button class="ui button ellipsis-button gt-mt-3 load-tags-and-branches" data-fetch-url="{{.RepoLink}}/commit/{{.CommitID}}/load-branches-and-tags" data-tooltip-content="{{.locale.Tr "repo.commit.load_referencing_branches_and_tags"}}" data-contained-in-text="{{.locale.Tr "repo.commit.contained_in"}}" aria-expanded="false">...</button>
```

Comment:
Why it is a "ellipsis-button" ?

Reply:
> Why is it an ellipsis-button?

https://github.com/go-gitea/gitea/pull/25180#issuecomment-1585041176

Reply:
But it's not right ..... there are JS for such button.

Reply:
Extracted `ellipsis-button` (CSS+JS) now into `ellipsis-button` (CSS) and `ellipsis-js-button` (JS).